apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/Trace/UnchainedTraceHeaderPropagatingHandler.cs

```diff
@@ -34,10 +35,18 @@ namespace Google.Cloud.Diagnostics.Common
     /// System.Net.Http.IHttpClientFactory defined in Microsoft.Extensions.Http.
     /// </para>
     /// </remarks>
+    // TODO: Marking as obsolete so that we can make this class internal. It cannot go away as this is now the correct
```

Comment:
I think you may need to move this comment above the XML comment, so that the XML comment is still associated with it.

Reply:
Ah yes, true, done